TAF_EasyRest/TestFramework/PageComponents/Owner/CreateMenuPageComponents/CreateListMenuEndPageComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+namespace TestFramework.PageComponents.Owner.CreateMenuPageComponents
+{
+    public class CreateListMenuEndPageComponent
+    {
+            private IWebDriver driver { get; }
+            public CreateListMenuEndPageComponent(IWebDriver driver)
+            {
+                this.driver = driver;
+            }
+
+            private IWebElement _finishButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[2]"));
+            private IWebElement _backButton => driver.FindElement(By.XPath("//p[contains(text(), 'Try out')]/parent::div/following-sibling::div[1]"));
+            public CreateListMenuEndPageComponent ClickFinishButton()
```

Comment:
At this step, you get a new component on the leftside menu (concretely a new menu item). You should define this somehow.